### PR TITLE
Initial stub for the GLB server

### PR DESCRIPTION
**Pull request by randallfrank**
Enable "one-shot" run mode.
Include the core classes for GLB ingest.

### CODE REVIEW

File: exts/ansys.tools.omniverse.dsgui/ansys/tools/omniverse/dsgui/extension.py [hunk 1]
```diff
@@ -128,7 +128,10 @@ def update_ui(self) -> None:
                 count = status.get("processed_buffers", 0)
                 total = status.get("total_buffers", 0)
                 dt = time.time() - status.get("start_time", 0.0)
-                tmp = f"Transfer: {count} of {total} : {dt:.2f}s"
+                percent = 0
```
Comment (randallfrank) [inline exts/ansys.tools.omniverse.dsgui/ansys/tools/omniverse/dsgui/extension.py:131]: Fix up the OV GUI to use percent

File: src/ansys/pyensight/core/exts/ansys.tools.omniverse.core/ansys/tools/omniverse/core/extension.py [hunk 2]
```diff
@@ -356,7 +371,9 @@ def help(self) -> None:
         self.warning("  --/exts/ansys.tools.omniverse.core/help=1")
         self.warning("     Display this help.")
         self.warning("  --/exts/ansys.tools.omniverse.core/run=1")
-        self.warning("     Run the server.")
+        self.warning("     Run the server until the DSG connection is broken.")
+        self.warning("  --/exts/ansys.tools.omniverse.core/run=0")
```
Comment (randallfrank) [inline src/ansys/pyensight/core/exts/ansys.tools.omniverse.core/ansys/tools/omniverse/core/extension.py:375]: Include "one-shot" mode that does a single DSG pull and conversation.  Useful for many things (e.g. "save as USD").

**Comment by randallfrank** [comment]
@mariostieriansys This work is not fully working, but it aligns my branch with the pyensight core and fixes a couple of things:

1) the error reported when a drive ran out of disk space
2) added a "one shot" DSG export mode
3) fixed the text in the dsgui module to use percentages

File: pyproject.toml [hunk 3]
```diff
@@ -35,6 +35,7 @@ dependencies = [
     "Pillow>=9.3.0",
     "pypng>=0.0.20",
     "psutil>=5.9.2",
+    "pygltflib>=1.16.2"
```
Comment (mariostieriansys) [inline pyproject.toml:38]: @randallfrank do we really want this as an hard dependency?
Shouldn't we put this as an hard dependency in the new repository?

Comment (randallfrank) [reply pyproject.toml:38]: @mariostieriansys let's reconsider in the context of the potential move to usd-core.

**Comment by randallfrank** [comment]
@david-bremer @mariostieriansys ok, this code is lining up for review.  It is missing tests yet, but it no longer uses 'kit's and aligns well with a pending EnSight PR for the UDT.   After this gets merged:

1) pytests need to be worked out (there are several ways we might do this.
2) need to bring the exts back to use this interfaace so that the OV extensions work again.
3) new documentation for this API.  Luckily, the utils.omniverse API did not change.

File: pyproject.toml (hunk 3 above)
Comment (randallfrank) [reply pyproject.toml]: Moved to ensight requirements_3_10.txt

File: exts/ansys.tools.omniverse.core/ansys/tools/omniverse/core/extension.py [hunk 4]
```diff
@@ -356,7 +375,9 @@ def help(self) -> None:
         self.warning("  --/exts/ansys.tools.omniverse.core/help=1")
         self.warning("     Display this help.")
         self.warning("  --/exts/ansys.tools.omniverse.core/run=1")
-        self.warning("     Run the server.")
+        self.warning("     Run the server until the DSG connection is broken.")
```
Comment (david-bremer) [inline exts/ansys.tools.omniverse.core/ansys/tools/omniverse/core/extension.py:378]: Do the old style options, with --/exts/... remain this way, for the kit extension?  And is that so we can continue to have an Omniverse GUI plugin?  

Or is this code that just hasn't been updated?

Comment (randallfrank) [reply exts/ansys.tools.omniverse.core/ansys/tools/omniverse/core/extension.py:378]: @david-bremer this PR breaks the extensions for now. Not too worried about this as OV will not see the change until we formally release again.   Yes, I need to replace that code, but I'd like to do so in another PR (as I want to get EnSight builds running against the rest of this code quickly).

**Review by david-bremer — DISMISSED** [review]

**Comment by randallfrank** [comment]
@david-bremer @mariostieriansys All checks/tests passed.  Just need a review.

**Review by david-bremer — APPROVED** [review]

**Pull request merged**